.github/dependabot.yml

```diff
@@ -122,17 +122,14 @@ updates:
         patterns:
           - '@angular/*'
           - '@angular-devkit/*'
+          - 'typescript'
           - 'zone.js'
       dependencies:
         patterns:
           - '*'
     ignore:
-      # Dependencies below are ordered alphabetically!
-      # Most/all Angular-related dependencies are ignored for all versions
-      - dependency-name: '@types/node'
-        update-types: ['version-update:semver-major']
```

Comment:
We want to keep the types in line with the actual Node.js-version we use, so it should not upgrade beyond v20 (at the moment).

There is unfortunately no way to say: `"ignore if version < .node-version"`